src/pages/scheduler/components/SchedulerCalendar.tsx

```diff
@@ -83,7 +83,9 @@ export const SchedulerCalendar = ({ term, courseCalendarEvents = [] }: Scheduler
       eventPropGetter={eventPropGetter}
       slotPropGetter={slotPropGetter(mode)}
       components={{
-        toolbar: CalendarToolBar(setSelectedDate, term || getCurrentTerm(), vCalendar),
+        toolbar: isMobile
+          ? MobileToolBar(setSelectedDate, term || getCurrentTerm(), vCalendar)
+          : CalendarToolBar(setSelectedDate, term || getCurrentTerm(), vCalendar),
```

Comment:
Could the `term` value be defined above to be defined. I think it'll require some object destruct renaming but it would clean this part up a lot. If we always default back to `getCurrentTerm` you could set the default value of that prop with it. 

Not an issue introduced from this PR but it was emphasized so just pointing it out.